### PR TITLE
TDKN-90:Move SSLContextProvider to daikon

### PR DESCRIPTION
**Pull request by wchen-talend**
**Please check if the PR fulfills these requirements**

- [x] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format) ?
- [x] Tests for the changes have been added (for bug fixes / features) ?
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The code coverage on new code >75%
- [ ] The new code does not introduce new technical issues (sonar / eslint)

**What kind of change does this PR introduce?**

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build / CI related changes
- [ ] Other... Please describe:

**What is the current behavior?** (You can also link to an open issue here)

No common ssl keystore/truststore loader.

**What is the new behavior?**

Add a new SSLContextProvider.java which can be used for both component and studio.

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

**Comment by build-talend** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/daikon_pull-request_master/233/) [:star:](https://ci-common.datapwn.com/job/daikon_pull-request_master/233/)

**Comment by build-talend** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/daikon_pull-request_master/239/) [:star:](https://ci-common.datapwn.com/job/daikon_pull-request_master/239/)

**Comment by sgandon** [comment]
please fill out the template for this PR if you need to have it reviewed.

File: daikon/src/main/java/org/talend/daikon/security/SSLContextProvider.java [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+    public synchronized static SSLContext buildContext(String algorithm, String keypath, String keypass, String keytype,
+            String trustpath, String trustpass, String trusttype) {
+        try {
+            KeyManager[] kms = buildKeyManagers(keypath, keypass, keytype);
+            TrustManager[] tms = buildTrustManagers(trustpath, trustpass, trusttype);
+            SSLContext context = SSLContext.getInstance(algorithm);
+            context.init(kms, tms, null);
+            return context;
+        } catch (Exception e) {
```
Comment (sgandon) [inline daikon/src/main/java/org/talend/daikon/security/SSLContextProvider.java]: Could you avoid the big Exception catch and detail all the exceptions handled by this catch please.

**Comment by build-talend** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/daikon_pull-request_master/272/) [:star:](https://ci-common.datapwn.com/job/daikon_pull-request_master/272/)

**Pull request merged**